pysiaf/utils/tools2.py

```diff
@@ -0,0 +1,217 @@
+"""A collection of helper functions to support pysiaf calculations and functionalities
+
+Authors
+-------
+
+    - Colin Cox
+
+References
+----------
+
+
+
+"""
+
+import sys
+import numpy as np
+from math import *
+import copy
+import pylab as pl
+
+from astropy.table import Table
+
+from ..aperture import PRD_REQUIRED_ATTRIBUTES_ORDERED
+from .polynomial import poly, ShiftCoeffs, Rotate, FlipY, FlipX, rotate_coefficients, RotateCoeffs, triangle, invert
+from ..siaf import Siaf
+from pysiaf.aperture import DISTORTION_ATTRIBUTES
+
+def checkinv(A, B, C, D, order):
```

Comment:
`tools.py` now includes a function `compute_roundtrip_error` which I adapted from your `checkinv`. Please evaluate whether it is necessary to still include this `checkinv`.